doc/source/manual/xtrabackup_sst.rst

```diff
@@ -371,7 +371,8 @@ every version is tested against a specific |PXB| version only:
 * |PXC| 5.6 requires |PXB| 2.3
 * |PXC| 5.7 requires |PXB| 2.4
 
-Other combinations are not guaranteed to work.
+Compatibility of other combinations is not guaranteed, and using the
```

Comment:
As far as I understood from the issue description, the idea of the fix is to emphasis the possibility of the cluster malfunction in case of other combinations. I also think "will remove cluster" to be bad wording. May be "...will make cluster inoperable", or something like this?

Reply:
The impact would probably be on the joiner side, the node may not be able to join the cluster.